### DISCUSSION TITLE
Dashboard / 'Top Events' Panel

### DISCUSSION THREAD

**Discussion by drush** [plausible/analytics#2639] [Anything Else]
While Custom Events are recorded and can be used to define Goals, there is currently no transparency about what Custom Events are being recorded.  This feature request is to add a new Dashboard panel, almost in the exact form of Top Pages that shows the Top events over the same time period.  

Adding this, would help during the integration phase to confirm that new custom events are being recorded properly. In addition, well designed event taxonomies can provide useful intelligence to the analytics teams without requiring significant additional functionality.

**Comment by neave** [comment]
+1 for this too. I use events to log errors, and a way to show this data in a table in the dashboard would be ideal.
